screens/Login.tsx

```diff
@@ -35,19 +35,19 @@ const Login: React.FC<LoginProps> = ({ navigation }) => {
 
   const handleLoginPress = () => {
     navigation.navigate("Main");
-    // WebBrowser.openAuthSessionAsync(
-    //   `${authUrl}?redirect_uri=${redirectURL}`,
-    //   redirectURL
-    // )
-    //   .then((result) => {
-    //     if (result.type === "success") {
-    //       Linking.openURL(result.url);
-    //     }
-    //   })
-    //   .catch((err) => {
-    //     console.error("Failed to open URL:", err.message);
-    //     alert("Failed to open URL");
-    //   });
+  //   WebBrowser.openAuthSessionAsync(
```

Comment:
remove all these comments

Reply:
This is how the actual auth works to login. I commented it out for now because else I can open the app. I think we should keep it for when we actually get it working.